rw_fs_record.c

```diff
@@ -0,0 +1,78 @@
+/******************************************************************************
+ * Read/write data structure from/to the file
+ *
+ * Copyright Thinnect Inc. 2020
+ * @license MIT
+ *****************************************************************************/
+#include "loglevels.h"
+#define __MODUUL__ "SNST"
+#define __LOG_LEVEL__ (LOG_LEVEL_SeqNumStorage & BASE_LOG_LEVEL)
+#include "log.h"
+
+#include <stdint.h>
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "fs.h"
+
+#define FILE_INST 0
```

Comment:
FILE_INST is not configurable.  I think this was supposed to be like a partition identifier and 0 is usually one of the 2 OTA related things. In most operating scenarios, 2 is used for user data. However the example does use 0 for fs. Long story short, it must at least be configurable, however other FS APIs take this as a parameter.